e2e/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "socket_example_client",
+  "version": "1.0.0",
+  "description": "Test that environment works",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "parcel index.html"
```

Comment:
Mmm... could it be a good idea to just add a command to launch the server and a command to launch the sample application?

Reply:
Ok. Starting both process from e2e, I think it's our best option, don't you?

Reply:
I'm thinking about consuming only the api from my front end, I would only want to launch the backend ip not the e2e / test runner app

Reply:
They are segregated process, I will add a start on redux-chat-back/package.json, what I meant that for simplicity to test e2e, will run both.

Reply:
Agree, to launch the e2e is a good idea to launch both, for me to consume it from the final app I would need a command to do it so (if it makes sense to wrap the node ... command), another option is to add in the readme.md instructions, to lunch just the api . call ..., to lunch the e2e + api call...